src/null-safety/index.md

```diff
@@ -135,22 +133,22 @@ listed by the analyzer.
 
 ### Dart 3 backwards compatibility
 
-Packages and apps migrated to use null safety with Dart 2.12 or later will
-likely be backwards compatible with Dart 3. Specifically, for any package where
-the lower bound of the SDK constraint is 2.12.0 or higher, pub will allow
-resolution even when the upper bound is limited to versions below 3.0.0. For
-example, a package with the following constraint will be allowed to resolve with
-a Dart 3.x SDK:
+Packages and apps migrated to use null safety with Dart 2.12 or later
+should be backwards compatible with Dart 3.
```

Comment:
question: Since the package was migrated with an older version, shouldn't this technically read "forwards compatible"?

```suggestion
should be forwards compatible with Dart 3.
```